### DISCUSSION TITLE
ULL-IACTEC Integration of USB Infrared & Ultraviolet Cameras

### DISCUSSION THREAD

**Discussion by fmarcanoull** [PlusToolkit/PlusLib#1045] [Show and tell]
Hi,
     Last week, during NA-MIC 2023 Las Palmas - Spain,  I've sent some pull requests for the integration of two sensors (Thermal Expert EV2 & PCO Ultraviolet cameras) to PTK. The requests were made for PlusBuild, PlusLib, PlusApp & PlusLibData (ULL-IACTEC branch). This code is currently operative for these sensors. I'd appreciate if any feedback could be received to know if there is any problem to be fixed and make this first version public.   

Bests,

Francisco J. Marcano
fmarcano@ull.edu.es
Universidad de La Laguna

**Comment by lassoan** [comment]
Could you please provide links to your pull requests? Since each pull request has a space for discussion, we can then continue the discussion in those pull requests.

**Comment by fmarcanoull** [comment]
Good morning Andras,

    Pull request URL’s are the following:

               PlusBuild:           https://github.com/PlusToolkit/PlusBuild/pull/89

               PlusLib:                https://github.com/PlusToolkit/PlusLib/pull/1043

               PlusLibData:        https://github.com/PlusToolkit/PlusLibData/pull/51

               PlusApp:               https://github.com/PlusToolkit/PlusApp/pull/77

 

Francisco.

 

 

De: Andras Lasso ***@***.***> 
Enviado el: lunes, 6 de febrero de 2023 19:38
Para: PlusToolkit/PlusLib ***@***.***>
CC: fmarcanoull ***@***.***>; Author ***@***.***>
Asunto: Re: [PlusToolkit/PlusLib] ULL-IACTEC Integration of USB Infrared & Ultraviolet Cameras (Discussion #1045)

 

Could you please provide links to your pull requests? Since each pull request has a space for discussion, we can then continue the discussion in those pull requests.
